worker_stack.go

```diff
@@ -62,7 +62,7 @@ func (wq *workerStack) refresh(duration time.Duration) []worker {
 
 func (wq *workerStack) binarySearch(l, r int, expiryTime time.Time) int {
 	for l <= r {
-		mid := int(uint(l+r) >> 1) // avoid overflow when computing mid
+		mid := l + ((r - l) >> 1) // avoid overflow when computing mid
```

Comment:
Please also add this comment `// avoid overflow when computing mid` to line 129 in `worker_loop_queue.go`, thanks!

Reply:
done!